### ISSUE TITLE
Turning off OTR for a peer causes bad behavior

### ISSUE THREAD

**Issue by olabini** [otrv4/pidgin-otrng#43]
If you inside of Pidgin says that OTR is turned off for ONE peer, you get weird behavior, primarily that someone starting an OTR conversation from the other side, still seems able to do it.

**Comment by olabini** [comment]
I'll take a look at this one.

**Comment by claucece** [comment]
Doing some investigation, it turns out that this happens:

```
Assertion failed: (client->client_profile != NULL), function otrng_client_get_client_profile, file client.c, line 833.
``` 

and, then, pidgin crashes.

**Comment by peterpalau** [comment]
> ```
> Assertion failed: (client->client_profile != NULL), function otrng_client_get_client_profile, file client.c, line 833.
> ```

This happens also when the server is not configured well.

I found this bug with @Cris2760 when we was reviewing another issue.

**Comment by claucece** [comment]
Yeah, it is caused by this: https://github.com/otrv4/pidgin-otrng/issues/107
The publication is not handled in the best way when locally something is not changed.
